app/controllers/subscriptions_controller.rb

```diff
@@ -122,4 +119,21 @@ def token
   def vacancy_id
     params.permit(:vacancy_id)[:vacancy_id]
   end
+
+  def updating_frequency?
+    params[:subscription].present?
```

Comment:
Do I need to implement some kind of strong params pattern here?

Reply:
No, you only need that when you're passing the params into something, so that people can't craft params that change fields you don't want changing.  In this case you're just inspecting a param so it's fine.